core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html

```diff
@@ -978,6 +980,11 @@ <h2>Suggestions to review</h2>
 
   <script src="/templates/dev/head/services/AssetsBackendApiService.js"></script>
   <script src="/templates/dev/head/services/StateTopAnswersStatsBackendApiService.js"></script>
+  <script src="/templates/dev/head/services/ValidatorsService.js"></script>
+  <script src="/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactory.js"></script>
+  <script src="/templates/dev/head/domain/classifier/ClassifierObjectFactory.js"></script>
+  <script src="/templates/dev/head/domain/exploration/ContentIdsToAudioTranslationsObjectFactory.js"></script>
```

Comment:
Why adding these three files if you didn't remove them from base.html?

Reply:
its not added by me.Its added by someone on develop branch.
When i merged it then it comes.

Reply:
You're merging it incorrectly then, because `AnswerClassificationResultObjectFactory.js` is duplicate of line 972, `ClassifierObjectFactory.js` of line 973 and `ContentIdsToAudioTranslationsObjectFactory.js` on line 990.

Reply:
Just remove the imports for these three files you added and we are good to go.

Reply:
ok sorry for that.
done